src/python/bot/fuzzers/libFuzzer/stats.py

```diff
@@ -288,8 +294,7 @@ def parse_performance_features(log_lines, strategies, arguments):
   # new_edges and new_features may not be correct when either corpus subset or
   # random max length strategy is used. Skip recording these stats in such case.
   # See https://github.com/google/clusterfuzz/issues/802.
-  if (not stats['strategy_corpus_subset'] and
-      not stats['strategy_random_max_len']):
+  if ('corpus_subset' not in strategies and 'random_max_len' not in strategies):
```

Comment:
aren't these always initialized to 0 ? did something change there ?

Reply:
I think that's no longer necessary now that we're no longer using autodetection.